bld/namelist_files/namelist_defaults_pop.xml

```diff
@@ -1333,6 +1333,12 @@
 <abio_atm_model_year>1</abio_atm_model_year>
 <abio_atm_data_year>1</abio_atm_data_year>
 
+<!--------------------------------->
+<!-- abio_dic_dic14 derived vars -->
+<!--------------------------------->
+
+<abio_dic_dic14_ltavg_NK>.false.</abio_dic_dic14_ltavg_NK>
```

Comment:
It seems like this variable serves the same function as `lecosys_tavg_all` (it's not passed to the Fortran code, it only exists so `build-namelist` and the `ocn.*.tavg.csh` script know what output to include in `tavg_contents`. Given that, should the variable name be structured similarly? I'd expect `labio_tavg_NK` or `labio_dic_dic14_tavg_NK`.

Reply:
I prefer `abio_dic_dic14_ltavg_NK`, to keep with the convention of all `abio_dic_dic14` related nml vars having the module name as a prefix.

I suggest that, at some point, we unify all ecosys related nml vars to have an `ecosys_` prefix, including changing `lecosys_tavg_all` to `ecosys_ltavg_all`. I think that is well beyond the scope of this PR.

Reply:
I opened #37 to track this thread, as I agree it's beyond the scope of this PR. I'm on the fence about whether to actually address the issue given the pending retirement of POP in CESM, but that's a discussion for the new issue ticket :)